<launch>
    <machine name="host" address="localhost">
    <machine name="raspi" address="rpi2-daisuke.local" env-loader="/home/ubuntu/work/ros/remote.sh" user="ubuntu" password="ubuntu" default="True" />
   <!-- joy node -->
    <node machine="host" pkg="joy" type="joy_node" name="teleopJoy" output="screen">
      <param name="dev" type="string" value="/dev/input/js2" />
      <param name="deadzone"          value="0.12" />
      <param name="autorepeat_rate"   value="1.0" />
      <param name="coalesce_interval" value="0.0" />
    </node>

    <!-- define buttons & Axes for HORI Pad/PS3 Pad-->
    <param name="axis_linear"   value="3"   type="int" />
    <param name="axis_angular"  value="0"   type="int" />
    <param name="scale_linear"  value="20.0" type="double" />
    <param name="scale_angular" value="80.0" type="double" />

    <node machine="host" pkg="rc_joystick" type="rc_joystick" name="rc_joystick" 
	launch-prefix="xterm -font r14 -bg darkblue -geometry 113x30+503+80 -e"
	output="screen" required="true" 
    />	
    <node machine="raspi" pkg="rosserial_python" type="serial_node.py" name="serial_node">
      <param name="port" value="/dev/ttyACM0"/>
    </node>
</launch>
